### ISSUE TITLE
fact update parameter problem

### ISSUE THREAD

**Issue by breu** [rcbops/opencenter#449]
Fact updating requires three positional arguments which are not obvious

root@breu-opencenter-server:~# opencentercli fact list | grep cpu_allocation_ratio
26 |16                            |NovaCluster             |cpu_allocation_ratio          |

updating - broken:
root@breu-opencenter-server:~# opencentercli fact update 26 cpu_allocation_ratio 32
No fact found for id or name cpu_allocation_ratio

updating - working:
root@breu-opencenter-server:~# opencentercli fact update 26 26 32

**Comment by hughsaunders** [comment]
fixed in 541a221da5b09b9913bc4e24d5d7809d7a08d647
There shouldn't have been an 'id_or_name' argument as fact id is enough to identify a fact.

**Comment by hughsaunders** [comment]
modified version: 

```
(default27)imac27:opencenterclient hms$ python shell.py fact update --help
usage: opencentercli fact update [-h] [--debug] [--endpoint ENDPOINT]
                                 fact_id value

positional arguments:
  fact_id              ID of fact to update
  value                new value

optional arguments:
  -h, --help           show this help message and exit
  --debug              Print debug information such as API requests
  --endpoint ENDPOINT  URL to opencenter endpoint. Should be of the form
                       http://host:8080 or https://user:pass@host:8443
```
